tests/wake-format/basic/stdout

```diff
@@ -1,168 +1,12 @@
-        sources "." `.*`
-    def srcs =
-        allSources
-        | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
-        | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
-        | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
-    def Pair testPaths wakePaths = splitBy (matches `tests/.*` _.getPathName) srcs
-    def wakeSourcesString =
-        wakePaths
-        | map (_.getPathName.format)
-        | foldr (_, ",\n  ", _) Nil
-        | cat
-    def testSourcesString =
-        testPaths
-        | map (getPathName _ | format | replace `^"tests/` '"{here}/')
-        | foldr (_, ",\n  ", _) Nil
-        | cat
-    def manifestStr = "%
-        # Generated by 'wake tarball Unit':
-        package build_wake
-        from wake import source , Nil
-
-        publish releaseAs = '%{release}', Nil
-        publish releaseOn = '%{time}', Nil
-        publish source =
-          %{wakeSourcesString}Nil
-        %"
-    def testsManifestStr = "%
-        # Generated by 'wake tarball Unit':
-        package test_wake
-        from wake import source , Nil
-
-        publish source =
-          %{testSourcesString}Nil
-        %"
-    # Create a manifest which describes the release and source files
-    require Pass manifest =
-        write "manifest.wake" manifestStr
-    require Pass testManifest =
-        write "tests/manifest.wake" testsManifestStr
-    # Execute tar to create a tarball of manifest + sources
-    require Pass tarball =
-        def cmd =
-            def gnutar = which "gnutar"
-            def tar = if gnutar ==* "gnutar" then which "tar" else gnutar
-            def files = map getPathName (manifest, testManifest, spec, srcs) | sortBy (_<*_)
-            tar, "--mtime={time}", "--transform=s@^@wake-{release}/@",
-            "--owner=0", "--group=0", "--numeric-owner", "-cJf",
-            "wake_{release}.tar.xz", files
-        job cmd (manifest, testManifest, spec, srcs)
-        | getJobOutput
-    Pass (tarball, changelog, Nil)
-
-export def static _: Result Path Error =
-    def filesResult = doInstall "tmp" "static"
-    def releaseResult = buildAs Unit
-    def timeResult = buildOn Unit
-    require Pass release = releaseResult
-    require Pass time = timeResult
-    require Pass files = filesResult
-    def cmd =
-        which "tar", "--mtime={time}", "--transform=s@^tmp/@wake-{release}/@",
-        "--owner=0", "--group=0", "--numeric-owner", "-cJf",
-        "wake-static_{release}.tar.xz", map getPathName files | sortBy (_<*_)
-    job cmd files
-    | getJobOutput
-
-from test_wake import topic wakeTestBinary
-from test_wake import topic wakeUnitTestBinary
-publish wakeTestBinary = defaultWake, Nil
-publish wakeUnitTestBinary = buildWakeUnit, Nil
-
-def defaultWake Unit =
-    require Pass wakeVisible =
-        doInstall "tmp" "default"
-
-    Pass (Pair "tmp/bin/wake" wakeVisible)
+def·name·Unit·=⏎
```

Comment:
Is the plan to eventually go back to spaces and regular newlines?

Reply:
Yeah, the formatter is in this weird middle state where some spaces/newlines are emitted by the formatter and some are from the original source. This helps to tell which is which. Once the formatter is 'done' all the spaces/newlines should be emitted by the formatter and none from the original source but we aren't quite there yet.

In the next PR, I'm moving these behind the debug flag